snow/consensus/snowball/network_test.go

```diff
@@ -26,34 +28,44 @@ func (n *Network) Initialize(params Parameters, numColors int) {
 	}
 }
 
-func (n *Network) AddNode(sb Consensus) {
+func (n *Network) AddNode(newConsensusFunc newConsensusFunc) Consensus {
```

Comment:
Is it possible for the caller to just give us an initialized `Consensus` instance instead of this function result?